pkgs/by-name/me/metadata/package.nix

```
@@ -0,0 +1,64 @@
+{ clang
+, libclang
+, stdenv
+, lib
+, fetchFromGitHub
+, pkg-config
+, ffmpeg_7
+, rustPlatform
+, asciidoc
+, glib
+, llvmPackages
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "metadata";
+  version = "0.1.9";
+
+  src = fetchFromGitHub {
+    owner = "zmwangx";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-OFWdCV9Msy/mNaSubqoJi4tBiFqL7RuWWQluSnKe4fU=";
+  };
+
+  cargoHash = "sha256-F5jXS/W600nbQtu1FD4+DawrFsO+5lJjvAvTiFKT840=";
+
+  nativeBuildInputs = [
+    pkg-config
+    asciidoc
+    clang
+    libclang.lib
+    llvmPackages.libcxxClang
+    clang
+  ];
+
+  postBuildHook = "a2x --doctype manpage --format manpage man/metadata.1.adoc";
+  postInstallHook = ''
+    mkdir -p $out/share/man
+    cp man/metadata.1 $out/share/man/
+  '';
+
+  buildInputs = [
+    ffmpeg_7
+    clang
+    glib
+    llvmPackages.libclang
+    stdenv.cc.libc
+  ];
+
+  env.LIBCLANG_PATH = "${libclang.lib}/lib";
+  env.FFMPEG_DIR = ffmpeg_7.dev;
+
+  meta = {
+    description = "Media file metadata for human consumption";
```

```suggestion
    description = "Media metadata parser and formatter designed for human consumption, powered by FFmpeg";
```

No need to add `longDescription`.